client/scripts/directives/field.js

```diff
@@ -212,13 +212,33 @@ require('../app').directive('field', /* @ngInject */function ($q, Raven, geoloca
 
         case 'select':
         case 'checkbox-group': {
+          // set viewModel as empty string for default value, otherwise typeahead filter is not working
+          field.viewModel = ''
+          $scope.$watch('field.model', function () {
+            if (!field.model) {
```

Comment:
this will catch when `field.model === 0`
```suggestion
            if (field.model == null || field.model === '') {
```